scripts/generate_code.py

```diff
@@ -507,6 +507,12 @@ def glob(path, pattern):
 # Python Reflection
 flatc_reflection(["-p"], "python/flatbuffers", "reflection")
 
+# Java Reflection
+flatc_reflection(
+    ["-j", "--java-package-prefix", "com.google.flatbuffers"],
+    "java/src/main/java", "com/google/flatbuffers/reflection"
```

Comment:
is there a reason it stored in such a deeply nested `com/google/flatbuffers/reflection` directory?

Reply:
My intent is to align with the package name of `com.google.flatbuffers.reflection` the same way the core api is in the package `com.google.flatbuffers` and the associated `*.java` files are in `java/src/main/java/com/google/flatbuffers`.  This is also consistent with the Python API where the generated reflection module, `flatbuffers.reflection`, is nested under the top level `flatbuffers` module.

Reply:
This also models the directory structure of the compiled class files when building `flatbuffers-XX.YY.XX.jar` maven via `mvn package` in the top level `java/` directory.